lessons/networking/networking-networking-hardware.md

```diff
@@ -1 +1,89 @@
+6. [Firewall](#firewall)
+
+# Lesson
+## Home Router Components
+Home routers typically contain following components:
+* Switch (Ethernet LAN)
+* Router (Ethernet WAN)
+* Broadband Connection
+* Wireless Access Point (Wi-Fi)
+* Firewall 
+
+It is important to note that these are typical components, but a router can have more or less depending on the specific application of the device. 
+
+![image](https://user-images.githubusercontent.com/18094862/179405675-5a1c6c1b-4f70-4b8e-89a4-982c522729c4.png)
+
+## Switch
+Switches facilitate sharing of resources on a Local Area Network (LAN) including all of the computers, printers, and servers. All of the connected devices can share information and usually these are all of the devices you own and trust. Ethernet cables are used to connect devices on the network. 
+
+![image](https://user-images.githubusercontent.com/18094862/179407510-eecacf84-1513-4565-a9e4-55acc99d9fc0.png)
+
+## Router
+Routers connect to an Internet Service Provider (ISP) which is part of the Wide Area Network (WAN). ISP connects various routers together so that they can all communicate. 
+
+Depending on the ISP, various different forms broadband connections are used which is described in next section.
+
+ISP networks contain routers just like home routers. The goal of the router is to direct traffic and choose the most efficient route to travel across the network including the internet. 
+
+![image](https://user-images.githubusercontent.com/18094862/179405602-b95cc6ef-db56-4b31-93b6-8354b759b69c.png)
+
+## Broadband Connection
+Broadband connection refers to the high-speed internet access technology which the ISP provides. 
+
+* Cable Modem
+  * Cable modem service uses coaxial cables that deliver picture sand sound to TV.
+* Digital Subscriber Line (DSL)
+  * DSL service uses traditional copper telephone lines.
+* Fiber
+  * Fiber technology uses light to carry data through transparent glass fibers
+* Wireless
+  * Wireless service uses radio link between home location and nearest service provider facility
+* Satellite
+  * Satellite service is a type of wireless broadband, but the communication is with a satellite rather than a nearby facility
+
+## Wireless Access Point
+Wireless Access Point component of the router provides Wi-Fi access to all of the devies on your local network without the need for Ethernet cables. 
+
+## Firewall 
+Home routers also contain a firewall which appropriately protects devices on your LAN from devices on the internet. If you have ever played games on consoles, some of them require modification of the home router firewall in terms of allowing certain ports through or setting up port forwarding which allows devices on the internet to connect to the console on your local LAN. 
+
+# Check YoSelf
+## Q1 - What component connects devices on a Local Area Network?
+a. ```firewall```<br>
+b. ```connector```<br>
+c. ```switch``` <---<br>
+d. ```hat```<br>
+
+## Q2 - What does WAN stand for?
```

Comment:
Try to have questions that are less about memorization and more about concepts. For example, if the answer is a fact they can easily look up in google, the question may not be that thought provoking

Reply:
Added more of conceptual questions